index.rst

```diff
@@ -318,56 +318,101 @@ being performed to clarify the classifications of data and services together.
 | Notebook               | execute                | high                   | Notebook               |
 +------------------------+------------------------+------------------------+------------------------+
 
+
+From these data classifications, a set of capabilities can be extrapolated. These capabilities
+can be expressly checked for authorization to the respective services. When a user first logs in,
+we may also map a list of all possible capabilities a user may have by checking group membership
+for a given instance. We can do this by programmatically constructing LDAP groups and adding
+users to that group. A group that is defined for this explicit purpose is called a capability group.
+**Membership in a capability group determines the possible capabilities a user may have.** All
+groups at NCSA must be namespaced, so there is a predefined prefix that exists for each
+capability group. Since that prefix would be instance-dependent (such as ``lsst_int_lsp_stable_``),
+we document the suffix for that capability.
+
++------------------------+------------------------+------------+-------------------------+
+| Resources              | Capabilities           | API Access | Capability Group Suffix |
++========================+========================+============+=========================+
+| Image Access           | read:image             | Yes        | img                     |
++------------------------+------------------------+------------+-------------------------+
+| Image Access           | read:image/md          | Yes        | img_md                  |
+| (Metadata)             |                        |            |                         |
++------------------------+------------------------+------------+-------------------------+
+| Table Access (DR,      | read:tap               | Yes        | tap                     |
+| Alerts)                |                        |            |                         |
++------------------------+------------------------+------------+-------------------------+
+| Table Access           | read:tap/efd           | Yes        | tap_efd                 |
+| (Transformed EFD)      |                        |            |                         |
++------------------------+------------------------+------------+-------------------------+
+| Table Access (User and | read:tap/user,         | Yes        | tap_usr                 |
+| Shared)                | write:tap/user         |            |                         |
++------------------------+------------------------+------------+-------------------------+
+| User Query History     | read:tap/history       | Yes        | tap_hist                |
++------------------------+------------------------+------------+-------------------------+
+| File/Workspace Access  | read:workspace         | Yes        | ws                      |
```

Comment:
I'm not sure there's ever going to be non-user file/workspace, but leave this in.

Reply:
Right now `/datasets` is not a non-user file/workspace, for example, it's LSST public data/data release data. We could rename it something else, but in terms of API it's served by the "workspace".

The intent is that you could give access to files from the project via webdav/VOSpace without giving access to your personal files.